### ISSUE TITLE
Name Property for 'update channel' Combo box in 'preferences' window is not defined 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#354]
### Steps to Reproduce

1. Launch Xamarin Workbook application.
2. Navigate to 'Tools' menu item present in menu bar and Expand it
3. Navigate to 'Preferences' menu item and select it
4. Navigate to 'update' tab item and select it
5. Navigate to update channel' Combo box and Observe Inspect property 

### Expected Behavior
Name property should be defined properly for Combo box (For e.g. 'update channel')

### Actual Behavior

Name property is not defined for Combo-box
**Narrator Behavior:** Narrator is not announcing  name property for Combo box and only announcing list item selected in Combo box as  'Stable, combo box, collapsed '
**NVDA Behavior:** NVDA is not announcing  name property for Combo box and only announcing list item selected in Combo box as  'Combo box,Stable,  collapsed '

### User Impact
Users who rely on screen reader will not be able to know the Name of combo box

### Screenshots
![update_channel_combobox_name_inspect](https://user-images.githubusercontent.com/22956130/40723801-49465ad0-643d-11e8-8efe-21c3d2259139.png)
![update_channel_combobox_name_narrator](https://user-images.githubusercontent.com/22956130/40723809-4c1c0a98-643d-11e8-881e-1a3b74cf2eb4.png)
![update_channel_combobox_name_nvda](https://user-images.githubusercontent.com/22956130/40723811-4e07a790-643d-11e8-8723-905672ec1f9d.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1
